src/gui/screen_lan_settings.c

```diff
@@ -5,7 +5,7 @@
  *      Author: Migi
  */
 
-//#define STATIC_SAVE_LOAD_CONFIG
+#define STATIC_SAVE_LOAD_CONFIG
```

Comment:
this macro define may be unnecessary? this feature is not any more optional i guess?